dialect/sql/schema/migrate.go

```diff
@@ -164,7 +181,7 @@ func (m *Migrate) create(ctx context.Context, tx dialect.Tx, tables ...*Table) e
 	// Create foreign keys after tables were created/altered,
 	// because circular foreign-key constraints are possible.
 	for _, t := range tables {
-		if len(t.ForeignKeys) == 0 {
+		if !m.withForeighKeys || len(t.ForeignKeys) == 0 {
```

Comment:
This loop creates the FKs. Let's skip it if the flag is false.

Reply:
skipped.